f8a_worker/models.py

```diff
@@ -111,13 +115,15 @@ def by_name(cls, session, name):
 class Package(Base):
     __tablename__ = 'packages'
     # ecosystem_id together with name must be unique
-    __table_args__ = (UniqueConstraint('ecosystem_id', 'name', name='ep_unique'),)
+    __table_args__ = (UniqueConstraint(
```

Comment:
Please don't mix new code and formatting changes in a single commit. It makes it harder to review. Ok for now though 😉